### PR TITLE
Performance tests

### PR DESCRIPTION
**Pull request by nahguam**

### CODE REVIEW

File: corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileProfileTest.java [hunk 1]
```diff
@@ -1,74 +0,0 @@
...
-    OrcFile orcFile = OrcFile.sink().schema(structTypeInfo).build();
-    Tap<?, ?, ?> tap = new Hfs(orcFile, temporaryFolder.getRoot().getCanonicalPath());
-
-    when(flowProcess.getConfigCopy()).thenReturn(conf);
-
-    @SuppressWarnings("unchecked")
-    TupleEntryCollector collector = tap.openForWrite(flowProcess);
-
-    long start = System.currentTimeMillis();
-    Tuple tuple = new Tuple("hello");
```
Comment (patduin) [inline corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileProfileTest.java]: Can we add some more columns, mixed types?

File: corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileSourcePerformanceTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  @Test
+  public void exerciseScheme() throws IOException {
+    TupleEntryIterator iterator = tap.openForRead(flowProcess);
+    while (iterator.hasNext()) {
+      iterator.next();
+    }
+    iterator.close();
+  }
+
+  private StructTypeInfo createTypeInfo() {
```
Comment (patduin) [inline corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileSourcePerformanceTest.java]: hey this looks familiar, maybe add a composite helper class or push it into some abstract super class??

Comment (nahguam) [inline corc-cascading/src/test/java/com/hotels/corc/cascading/OrcFileSourcePerformanceTest.java]: I thought, for the sake of simplicity and readability there's no harm in a little duplication in a test.

File: pom.xml [hunk 3]
```diff
@@ -371,6 +370,54 @@
 
   <profiles>
     <profile>
+      <id>unit-test</id>
```
Comment (patduin) [inline pom.xml]: so by default surefire includes:
"\*\*/Test\*.java" - includes all of its subdirectories and all Java filenames that start with "Test".
"\*\*/\*Test.java" - includes all of its subdirectories and all Java filenames that end with "Test".
"\*\*/\*TestCase.java" - includes all of its subdirectories and all Java filenames that end with "TestCase".

So if we name our performance test, *TestPerformance it won't be picked up and we just need one profile
```xml
   <profile>
      <id>test-performance</id>
      <build>
        <plugins>
          <plugin>
            <artifactId>maven-surefire-plugin</artifactId>
            <configuration>
              <includes>
                <include>**/*TestPerformance.java</include>
              </includes>
            </configuration>
          </plugin>
        </plugins>
      </build>
    </profile>
```
Then running tests:
mvn test

Running performance tests + performance tests:
mvn test -Ptest-performance

We can no longer *only* run performance tests not sure if that is bad.

too tricksy??

**Pull request merged**